Jenkinsfile

```diff
@@ -14,9 +14,27 @@
  */
 
 /**
- * These are the platforms we are building against.
+ * These are global variables defining the
+ * parameterized platforms we are building against.
  */
-def platforms = ["linux", "mac", "windows"]
+def TEMPLATE_URLS = [:]
+def DCOS_VERSIONS = ["1.8.9", "1.9.2", "master"]
+
+withCredentials(
+    [[$class: 'StringBinding',
+      credentialsId: '1c2fb1a3-38a6-4289-a831-11da1b319412',
+      variable: 'CF_TEMPLATE_URL_1_8_9'],
+     [$class: 'StringBinding',
+      credentialsId: '542aa287-9464-49cb-bdd3-9e4ca457dd87',
+      variable: 'CF_TEMPLATE_URL_1_9_2'],
+     [$class: 'StringBinding',
+      credentialsId: 'fd1fe0ae-113d-4096-87b2-15aa9606bb4e',
+      variable: 'CF_TEMPLATE_URL_MASTER']]) {
+
+    TEMPLATE_URLS['1.8.9'] = "${env.CF_TEMPLATE_URL_1_8_9}"
+    TEMPLATE_URLS['1.9.2'] = "${env.CF_TEMPLATE_URL_1_9_2}"
+    TEMPLATE_URLS['master'] = "${env.CF_TEMPLATE_URL_MASTER}"
+}
```

Comment:
I've now renamed these to 'master', 'stable', and 'legacy'. For now these are pulling from `testing/master`, `stable`, and `testing/1.8.9`. We will need to define a permalink for `legacy` (or come up with a new name for this) in order to make this all automated. In the meantime, I will maintain the `legacy` URL manually.